CODENAMES.md

```diff
@@ -11,5 +11,11 @@ releases are subject to change.
 * Hydrogen (18.x 2022)
 * Iron (20.x 2023)
 * Jod (22.x 2024)
+* Krypton (24.x 2025)
+* Lithium (26.x 2026)
+* Magnesium (28.x 2027)
```

Comment:
Alternatives:

- Manganese
- Meitnerium
- Mendelevium
- Mercury
- Molybdenum
- Moscovium

Reply:
I'd suggest not picking "mercury" for the letter "m" -- one [potential pick for the letter "q"](https://github.com/nodejs/Release/issues/304#issuecomment-1625534993) is "quicksilver", which is an alternative name for "mercury". I may have intentionally stopped short of "q" in this PR 🙂.

Reply:
If 26.x can be **Lead**, 28.x could be **Molybdenum**, which means "lead-like" according to https://en.wikipedia.org/wiki/List_of_chemical_element_name_etymologies